spec/schemas/_common.mapping.yaml

```diff
@@ -843,20 +743,22 @@ components:
             - type
     GeoOrientation:
       oneOf:
+        # eslint-disable yml/sort-sequence-values
```

Comment:
Does the order here matter, otherwise remove `eslint-disable`.

Reply:
They're in preference order as left & right are the true values and cw/ccw are the aliases